apps/wallet/ios/Podfile.lock

```diff
@@ -841,6 +844,8 @@ EXTERNAL SOURCES:
     :path: "../../../node_modules/react-native/ReactCommon/runtimeexecutor"
   ReactCommon:
     :path: "../../../node_modules/react-native/ReactCommon"
+  RNCAsyncStorage:
+    :path: "../../../node_modules/@react-native-async-storage/async-storage"
```

Comment:
after reviewing you storage function, remove these changes if it is not needed anymore